source/reference/mongosync.txt

```diff
@@ -32,6 +32,8 @@ line options.
    - Standalone MongoDB instances are not supported. :ref:`Convert the
      standalone instance <server-replica-set-deploy-convert>` to a
      replica set before using {+c2c-product-name+}. 
+   - For more information on ``mongosync`` limitations,
```

Comment:
[suggestion] It might be helpful for the user to just list out the limitations here and _then_ link out to the limitations, rather than directing them to another page with little context!
```suggestion
   - ``mongosync`` does not support Atlas shared clusters or Serverless instances. For more information, see :ref:`c2c-limitations`. 
```